### PR TITLE
Enables successful alsa+HDMI2USB capture

### PR DESCRIPTION
**Pull request by xfxf**
This has been used successfully for Pycon AU 2016.

Adds audiorate/audio queue,  adds a audio/video delay (100ms audio delay required on everything I've tested so far else sync is a few frames out).

Please merge!

### CODE REVIEW

**Comment by CarlFK** [comment]
This branch has conflicts that must be resolved

**Comment by xfxf** [comment]
@CarlFK resolved conflicts, this should be good to merge

File: ingest.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python3
-
-# ingest.py
 """
-Source client for Voctomix.
+ingest.py: source client for Voctomix.
+
+Ryan Verner <ryan@nextdayvideo.com.au>
+Carl Karsten <carl@nextdayvideo.com>
```
Comment (xfxf) [inline ingest.py:6]: TBH i don't care what order this in, feel free to juggle, just good to have this here IMO esp if it'll be part of voctomix's example scripts

**Comment by CarlFK** [comment]
I'm merging but undoing the $hostname bits.    That should go in the calling script. (or in my case, not in the calling script because I stopped trying to keep room and hostname in sync)

**Pull request merged**